docker-compose.dev.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+# Config file for registry settings in Azure App Service dev
```

Comment:
Hvordan blir denne brukt av Azure egentlig?

Reply:
La fila her for at man skal ha den et sted. Det er den som legges inn i App Service config for å spinne opp både Django og Celery.